### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Piumal1999**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+    
+    Go to the created application and click Generate button.
+    
+    Copy the key and secret to `sefwebapps/invoker/src/main/resources/configprops.properties` file
+    
+5. Build the invoker module using maven.
+    ```$xslt
+    mvn clean install
+    ``` 
+    Deploy the generated war in tomcat9 server.
```
Comment (jayasanka-sack) [inline README.md]: Move this step to under a new title. "Building the invoker"

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+## Building the project
+
+1. Build the project using maven
+    ```$xslt
+    mvn clean install
+    ```
+2. Deploy three war files in module target directories in tomcat9
+
```
Comment (jayasanka-sack) [inline README.md]: Remove this section

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+### How to use invoker to invoke APIs?
+1. To invoke any open endpoints use `http://localhost:8080/invoker/open/api/<api-url>`
+
+    ex: `http://localhost:8080/invoker/open/api/partnership/v1/engagements`
+    
+2. To invoke other endpoints use `http://localhost:8080/invoker/api/<api-url>`
+3. To login use:
+    `POST: http://localhost:8080/invoker/login`
+4. Import  `scripts/postman_collection.json` to postman to see available endpoints.
```
Comment (jayasanka-sack) [inline README.md]: Remove the 4th step for now. We don't have a postman collection for invoker yet.

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
 # sef-webapps
```
Comment (jayasanka-sack) [inline README.md]: Let's rename this to SEF Common Webapps

File: README.md (hunk 1 above)
Comment (Piumal1999) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Piumal1999) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Piumal1999) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (Piumal1999) [reply README.md]: Done

**Comment by Piumal1999** [comment]
Issue #16

**Pull request merged**